src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.task;
+
+public class Task {
+
+    protected String taskType;
+    protected String taskName;
+    protected String taskTime;
+    protected boolean isDone;
+    public static int numOfTasks = 0;
+
+    public Task(String description) {
+        this.taskName = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "V" : "X"); //return tick or cross
```

Comment:
The format for in-line comment is `// Return tick or cross.` There is a space after `//` and a `.` after the comment.